### ISSUE TITLE
CI is Failing Intermittently

### ISSUE THREAD

**Issue by ashfurrow** [Moya/Moya#259]
Not sure what caused it. Looks like a problem with Xcode (go figure).

```
The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(1 failure)
2015-10-22 00:38:04.847 xcodebuild[769:3065] [MT] IDETestOperationsObserverDebug: (357FE833-30CF-44F9-9DD3-7F464EB8F250) Beginning test session with Xcode 7A220 on target <DVTiPhoneSimulator: 0x7f99c2cdecf0> {
		SimDevice: SimDevice : iPhone 6 (CEBBD79A-B6C1-4A02-BD33-56BBCBDED03E) : state={ Shutdown } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPhone-6 } runtime={ SimRuntime : 9.0 (13A340) - com.apple.CoreSimulator.SimRuntime.iOS-9-0 }
} (9.0 (13A340))
2015-10-22 00:38:33.992 xcodebuild[769:4353] Connection peer refused channel request for "dtxproxy:XCTestManager_IDEInterface:XCTestManager_DaemonConnectionInterface"; channel canceled <DTXChannel: 0x7f99c33d1c20>
Testing failed:
	Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc failed with exit code 1
** TEST FAILED **
```

**Comment by vinnybad** [comment]
For what it's worth, I'm seeing the same thing happen on our jenkins instance.

**Comment by ashfurrow** [comment]
Misery loves company :)

**Comment by ashfurrow** [comment]
Hmm, we're actually not seeing this problem anymore. Going to close the issue.

**Comment by ashfurrow** [comment]
Looks like I spoke too soon.

**Comment by vinnybad** [comment]
I had this issue when multiple builds were executing at the same time. Decreasing the number of executors to 1 resolved the issue. Not an ideal solution but this bought us some time to look for other solutions.

**Comment by SRandazzo** [comment]
Little bit of snooping around led me to this issue on Travis-CI:
https://github.com/travis-ci/travis-ci/issues/4725

This appears to be a known bug. There is a potential workaround where you can a `sleep n` to the end of our travis.yml

Thoughts on doing the above hack?

**Comment by ashfurrow** [comment]
Can't hurt!

**Comment by ashfurrow** [comment]
Cool, so this is hopefully fixed in #317, but let's keep this issue open for a while to make sure.
